### PR TITLE
[1LP][RFR]Adding automation for Customer  BZ1644770

### PR DESCRIPTION
**Pull request by kedark3**
## Purpose or Intent

- __Adding tests for Customer BZ 1644770__ 

### PRT Run
{{pytest: cfme/tests/infrastructure/test_vmware_provider.py -k 'test_vm_remove_from_inventory_orphan or test_delete_vm_archive'  -v}}

### CODE REVIEW

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 1]
```diff
@@ -558,3 +558,49 @@ def test_vm_notes_ui(appliance, provider):
...
+    """
+    Bugzilla:
+        1644770
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: functional
+        testSteps:
```
[the remark this answers is not in the document]
Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: @john-dupuy I was told earlier that expected results are not needed for Automated tests. Does that not hold true? I am ok either way.

[the remark this answers is not in the document]
Comment (john-dupuy) [reply cfme/tests/infrastructure/test_vmware_provider.py]: I don't think testSteps or expected results are necessary for automated tests. But if you put the test steps in there, may as well put in the expected results.

[the remark this answers is not in the document]
Comment (kedark3) [reply cfme/tests/infrastructure/test_vmware_provider.py]: Done.

**Comment by kedark3** [comment]
@sshveta 1 out of 2 tests are failing in latest run 749 as it needs Wrapanapi PR RedHatQE/wrapanapi/pull/436  to be merged.

**Comment by mshriver** [comment]
@kedark3 wrapanapi 3.5.5 has been tagged, should be available on pypi shortly.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by kedark3** [comment]
@mshriver seems like pip didn't pick up latest package yet for wrapanapi release.

File: cfme/tests/infrastructure/test_vmware_provider.py [hunk 2]
```diff
@@ -558,3 +558,59 @@ def test_vm_notes_ui(appliance, provider):
...
+            2.Create a new Vm
+            3.Remove it from vmware using delete operation and check status of that vm in CFME -
+            should be archived
+        expectedResults:
+            1.VMware provider added successfully.
+            2.New VM visible in CFME
+            3.VM is Archived in CFME
+    """
+    create_vm.mgmt.delete()
+    create_vm.wait_for_vm_state_change(from_any_provider=True, desired_state='archived')
```
Comment (mshriver) [inline cfme/tests/infrastructure/test_vmware_provider.py]: Can we wrap this in a try/except to call `pytest.fail` on the TimedOutError exception please?

**Comment by dajoRH** [comment]
Lint report for commit 3562d3757dbeed0d231e46938055207f2569c625:

`cfme/tests/infrastructure/test_vmware_provider.py`:
- :red_circle: Line 620:12: **F821** *undefined name 'TimedoutError'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**